library/src/main/java/dev/chrisbanes/insetter/Insetter.kt

```diff
@@ -366,9 +366,11 @@ class Insetter private constructor(builder: Builder) {
         ViewCompat.setOnApplyWindowInsetsListener(view) { v, insets ->
             if (onApplyInsetsListener != null) {
                 // If we have an onApplyInsetsListener, invoke it
-                onApplyInsetsListener.onApplyInsets(v, insets, initialState)
-                // We don't know what sides have been applied, so we assume all
-                return@setOnApplyWindowInsetsListener WindowInsetsCompat.CONSUMED
+                return@setOnApplyWindowInsetsListener onApplyInsetsListener.onApplyInsets(
```

Comment:
Hmmm, I'd prefer we keep the existing API signature. Instead, lets use the `consume` value (like below):

```
return if (consume != CONSUME_NONE) insets
else WindowInsetsCompat.CONSUMED
```